Helper/Config.php

```diff
@@ -519,12 +519,14 @@ public function debugLogsEnabled($storeId): bool
 
     public function getAutoCaptureOpenInvoice(int $storeId): bool
     {
-        return $this->getConfigData('auto_capture_openinvoice', self::XML_ADYEN_ABSTRACT_PREFIX, $storeId, true);
+        $captureForOpenInvoice = $this->getConfigData('capture_for_openinvoice', self::XML_ADYEN_ABSTRACT_PREFIX, $storeId, true);
+        if($captureForOpenInvoice == 'auto') return true;
```

Comment:
Could you please use `strcmp()` for the comparison and assign the string `auto` to a constant?